modules/programs/ledger.nix

```diff
@@ -0,0 +1,196 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.programs.ledger;
+  package = pkgs.ledger;
+  boolOption = value: name: if value == true then "--${name}" else "";
+  valueOption = value: name: if value != null then "--${name} ${value}" else "";
+in {
+  meta.maintainers = [ maintainers.piperswe ];
+
+  options.programs.ledger = {
```

Comment:
Try to limit the options to [valuable ones](https://github.com/Infinisil/rfcs/blob/config-option/rfcs/0042-config-option.md#valuable-options) to reduce maintenance effort.